src/Grid/FeatureGrid/FeatureGrid.jsx

```diff
@@ -838,9 +843,15 @@ export class FeatureGrid extends React.Component {
       ? `${className} ${this._className}`
       : this._className;
 
-    const finalRowClassName = rowClassName
-      ? `${rowClassName} ${this._rowClassName}`
-      : this._rowClassName;
+    let rowClassNameFn;
+    if (isFunction(rowClassName)) {
+      rowClassNameFn = rowClassName;
```

Comment:
I think this ommits `this._rowClassName` which might not be intentional.

Reply:
Alright, I've created a function that calls passed function of prop and appends `this._rowClassName`. See 558dc531